Description: Keep only the shared hidraw backend due to problems in others
Last-Update: 2025-08-08
Author: Blair Noctis <ncts@debian.org>
Forwarded: not-needed
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -90,7 +90 @@
-    "illumos-static-libusb",
-]
-illumos-shared-libusb = []
-illumos-static-libusb = []
-linux-native = [
-    "dep:udev",
-    "dep:nix",
+    #"illumos-static-libusb",
@@ -97,0 +92,6 @@
+#illumos-shared-libusb = []
+#illumos-static-libusb = []
+#linux-native = [
+#"dep:udev",
+#"dep:nix",
+#]
@@ -99 +99 @@
-linux-shared-libusb = []
+#linux-shared-libusb = []
@@ -102 +102 @@
-macos-shared-device = []
+#macos-shared-device = []
@@ -115 +115 @@
-[target.'cfg(target_os = "linux")'.dependencies.nix]
+[disabled.target.'cfg(target_os = "linux")'.dependencies.nix]
@@ -124 +124 @@
-[target.'cfg(target_os = "linux")'.dependencies.udev]
+[disabled.target.'cfg(target_os = "linux")'.dependencies.udev]
